buildpacks/static-web-server/README.md

```diff
@@ -23,52 +22,144 @@ In the app source repo, add the buildpack to [`project.toml`](https://buildpacks
+const apiUrl = document.body.dataset.public_api_url || 'https://api.example.com';
+const response = await fetch(apiUrl, {
+  method: "POST",
+  // …
+});
+```
+
+Alternatively, default values can be preset in the HTML document's body element:
+
+```html
+<html>
+<head>
+  <title>Example</title>
+</head>
+<!-- If the PUBLIC_API_URL variable is set, this value in the document will be overwritten -->
+<body data-public_api_url="https://api.example.com">
+  <h1>Example</h1>
+</body>
+</html>  
 ```
 
+Then, the javascript does not need a default value specified.
+
+```javascript
+const response = await fetch(document.body.dataset.public_api_url, {
+  method: "POST",
+  // …
+});
+```
+
+## Build-time Configuration
+
+_Static config that controls how the app is built, and how the web server delivers it._
+
+This is set in the app source repo [`project.toml`](https://buildpacks.io/docs/reference/config/project-descriptor/) file and processed during CNB build. Rebuild is necessary to apply any changes.
+
 ### Static Build Command
 
 *Default: (none)*
 
-This buildpack also supports a executing a build command during CNB Build process. The output of this command is saved in the container image, and will not be re-built during release, versus the [Release Build Command](#release-build-command)).
+This buildpack supports a executing a build command during CNB Build process. The output of this command is saved in the container image.
+
+For apps built with Node.js, execution of the build command is typically handled automatically by [heroku/nodejs CNB's build script hooks](https://github.com/heroku/buildpacks-nodejs/blob/main/README.md#build-script-hooks), and does not need to be configured here.
+
+If your static web app is a static site generator built in a language other than JS, then you may need to configure the static site build command here. For example, [Hugo](https://gohugo.io) written in Go:
 
 ```toml
 [com.heroku.static-web-server.build]
 command = "sh"
-args = ["-c", "npm build"]
+args = ["-c", "hugo"]
 ```
 
 This static build command does not have access to Heroku app config vars, but still can be configured using CNB Build variables in `project.toml`:
 
 ```toml
  [[io.buildpacks.build.env]]
- name = "API_URL"
- value = "https://test.example.com/api/v7"
+ name = "HUGO_ENABLE_ROBOTS_TXT"
+ value = "true"
+```
+
+When dependent on another language's compiled program like this, ensure that the app's buildpacks are ordered with `heroku/static-web-server` last, after the language buildpack.
+
+```toml
+[[io.buildpacks.group]]
+id = "heroku/go"
+
+[[io.buildpacks.group]]
+id = "heroku/static-web-server"
+```
+
+### Runtime Configuration Enabled
+
+*Default: true*
+
+The [Runtime App Configuration](#runtime-app-configuration) feature may be disabled, such as when it is completely uneccesary or undesirable for a specific app.
+
+```toml
+[com.heroku.static-web-server.runtime_config]
```

Comment:
Missing `html_files` config?

Reply:
[`html_files` config](https://github.com/heroku/buildpacks-frontend-web/blob/env-as-html-data/buildpacks/static-web-server/README.md#runtime-configuration-html-files) is now documented 🙌